app/controllers/openstax/api/v1/api_controller.rb

```diff
@@ -75,7 +75,7 @@ def set_date_header
         def set_cors_preflight_headers
           headers['Access-Control-Allow-Origin'] = '*'
           headers['Access-Control-Allow-Methods'] = 'GET, HEAD, POST, PUT, PATCH, DELETE, OPTIONS'
-          headers['Access-Control-Allow-Headers'] = 'X-Requested-With, X-Prototype-Version, X-CSRF-Token, Token'
+          headers['Access-Control-Allow-Headers'] = 'X-Requested-With, X-Prototype-Version, X-CSRF-Token, Token, Authorization'
           headers['Access-Control-Max-Age'] = '86400'
         end
```

Comment:
I would be fine making these headers configurable in the sites that use openstax_api, with defaults as listed (or more restricted if you want and if generic restrictive defaults can be chosen).  The configurable options could be a proc so the sites could have custom logic.